gulpfile.js

```diff
@@ -204,6 +206,7 @@ gulp.task( 'customJS', function() {
 				]
 			})
 		)
+		.pipe( remember( 'customJS' ) ) // Bring all files back to stream
```

Comment:
Looks like you missed other periods.

Reply:
I think I got them all now. Some weren't related to this change, but figured we should fix them all.